tactics/btermdn.ml

```diff
@@ -22,6 +21,7 @@ let dnet_depth = ref 8
 
 type term_label =
 | GRLabel of GlobRef.t
+| ProjLabel of Projection.t
```

Comment:
compare_term_label below needs to be changed (Stdlib.compare isn't ok on ProjLabel)